validator/shelltests/help.test

```diff
@@ -0,0 +1,15 @@
+./otel_config_validator --help
```

Comment:
What's going on with these `./validator/shelltests` files?

Reply:
They are just tests of the compiled cli. From the readme, `shelltest -c --diff --all shelltests/*.test`.